assets/css/people.css

```diff
@@ -0,0 +1,76 @@
+body {
+    background-color: #FDFFF7;
```

Comment:
Could we store the color scheme colors as variables? It will make things easier in the future